### ISSUE TITLE
Add official orcaslicer profiles for Er-20 / Thinker / Star (all eryone printer)

### ISSUE THREAD

**Issue by Hangtimehippie** [Eryone/STM32#13]
I noticed that eryone github pulled a request to orcaslicer to implement the Thinker X400.
https://github.com/Eryone/OrcaSlicer

Would it also be possible according to the Er-20 / Thinker and SE / Star an official printer selection.

then at least all models would be openly involved in the orcaslicer.

**Comment by CoolGuyWithAPen** [comment]
I agree, trying to get the thinker se on orca slicer but I lost the printer profile
